nuntium/templates/nuntium/message/message_in_list.html

```diff
@@ -28,7 +28,10 @@
 	{% endfor %}
 
 	<br />
-	<span class="author_name col-md-4"><small>{% blocktrans with message.author_name as name %}Asked by {{ name }}.{% endblocktrans %}</small></span>
+	<span class="author_name col-md-4"><small>
+    {% url 'all-messages-from-the-same-author-as' slug=message.writeitinstance.slug message_slug=message.slug as other_messages_by %}
+    {% blocktrans with message.author_name as name %}Asked by 
+    <a href="{{other_messages_by}}">{{ name }}</a>.{% endblocktrans %}</small></span>
```

Comment:
Ditto here - `other_messages_by` needs to be in the with of the `blocktrans` unless I've misunderstood how blocktrans works (which I could well have).